modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -3,208 +3,131 @@ ifndef::env-site,env-github[]
-
-This configuration of the external load balancer:
+. From Anypoint Platform, select *Access Management*.
++
+If you do not have access to *Access Management*, ask your organization admin (system administrator) to provide the "Manage Runtime Fabrics" permission to your user. If you have access, execute the following steps to add the permission:
+. Select *Users*.
+. Locate the row containing your username and then select the blue link in the *Username* column.
+. In the *Runtime Manager* tab, enable the *Manage Runtime Fabrics* permission for your Runtime Fabric environment.
++
+[NOTE]
+It can take up to five minutes for any permissions you add to propagate.
 
-* Maintains high availability
-* Protects against failures
-* Gracefully handles automated failover if a replica of the internal load balancer restarts or is evicted and rescheduled on another controller VM.
+== Step 2: Configure Inbound Traffic Using Runtime Manager
 
+. Navigate to Runtime Manager and select *Runtime Fabric*.
+. Select the name of your Runtime Fabric to open the management page.
+. In the *Associated Environments* tab, ensure that your Runtime Fabric is associated with the environments in which applications are to be deployed.
+. Select *Inbound Traffic* and toggle *Enable inbound traffic* to on.
++
 [NOTE]
-Controller VMs are virtual machines dedicated to run the components which power Anypoint Runtime Fabric.
+If you disable the Inbound Traffic tab, changes can take several minutes to complete.
 
-All inbound traffic entering Anypoint Runtime Fabric is encrypted using TLS. The following procedures describe how to configure and use a TLS certificate to enable inbound traffic. Once enabled, HTTP requests sent to Runtime Fabric will receive a 301 response to redirect to HTTPS on port 443.
+. In the *Basic Configuration* section, verify that *Secure Port* is set to `443` and that *Non Secure Port* is set to `80`.  The nonsecure port redirects to the secure port.
+. In the *Resource Allocation* section:
++
+The number of inbound load balancer replicas is determined by the number of controller nodes configured in Runtime Fabric. A minimum of three replicas is required for production environments, and these replicas run only on the controller VMs. Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
++
+*Shared Mode* is the default setting if no dedicated load balancer node is added in Runtime Fabric. In this mode, the internal load balancer runs on all controller nodes with the amount of CPU and memory specified. *Dedicated Mode* specifies that the entire node is available, so you cannot specify the amount of cores and memory. If *Dedicated Mode* is disabled, then all nodes are already in sharing mode, which prevents them from being dedicated.
```

Comment:
Think we should split up shared and dedicated mode. It seems like there should be a break in between those options -- perhaps listed.

Reply:
Thank you - I've updated this.